src/test/java/de/tum/in/www1/artemis/connector/BambooRequestMockProvider.java

```diff
@@ -224,95 +215,32 @@ else if (buildPlanToBeReturned != null) {
-        mockServer.expect(requestTo(uri)).andExpect(method(HttpMethod.POST)).andRespond(withStatus(HttpStatus.OK));
-    }
-
-    @NotNull
-    public List<BambooTriggerDTO> mockGetTriggerList(String buildPlanKey) throws IOException, URISyntaxException {
-        var triggerList = List.of(new BambooTriggerDTO(1L, "foo", "artemis"));
-
-        MultiValueMap<String, String> parameters = new LinkedMultiValueMap<>();
-        parameters.add("buildKey", buildPlanKey);
-        final var triggerListHtmlResponse = loadFileFromResources("test-data/bamboo-response/build-plan-trigger-list-response.html");
-        URI uri = UriComponentsBuilder.fromUri(bambooServerUrl.toURI()).path("/chain/admin/config/editChainTriggers.action").queryParams(parameters).build().toUri();
-        mockServer.expect(requestTo(uri)).andExpect(method(HttpMethod.GET)).andRespond(withStatus(HttpStatus.OK).contentType(MediaType.TEXT_HTML).body(triggerListHtmlResponse));
-        return triggerList;
-    }
-
-    public void mockUpdateRepository(String buildPlanKey, BambooRepositoryDTO bambooRepository, BitbucketRepositoryDTO bitbucketRepository,
-            ApplicationLinksDTO.ApplicationLinkDTO applicationLink, String defaultBranch) throws URISyntaxException {
-        MultiValueMap<String, String> parameters = new LinkedMultiValueMap<>();
-        parameters.add("planKey", buildPlanKey);
-        parameters.add("selectedRepository", "com.atlassian.bamboo.plugins.stash.atlassian-bamboo-plugin-stash:stash-rep");
+        parameters.add("selectedRepository", "com.atlassian.bamboo.plugins.atlassian-bamboo-plugin-git:gitv2");
         parameters.add("repositoryName", bambooRepository.getName());
         parameters.add("repositoryId", Long.toString(bambooRepository.getId()));
         parameters.add("confirm", "true");
         parameters.add("save", "Save repository");
         parameters.add("bamboo.successReturnMode", "json");
-        parameters.add("repository.stash.branch", defaultBranch);
-        parameters.add("repository.stash.repositoryId", bitbucketRepository.id());
-        parameters.add("repository.stash.repositorySlug", bitbucketRepository.slug());
-        parameters.add("repository.stash.projectKey", bitbucketRepository.project().key());
-        parameters.add("repository.stash.repositoryUrl", bitbucketRepository.getCloneSshUrl());
-        parameters.add("repository.stash.server", applicationLink.getId());
+        parameters.add("repository.git.branch", defaultBranch);
+        parameters.add("repository.git.repositoryUrl", newRepoUrl);
+        parameters.add("repository.git.authenticationType", "PASSWORD");
+        parameters.add("repository.git.passwordCredentialsSource", "SHARED_CREDENTIALS");
+        parameters.add("repository.git.useShallowClones", "true");
+        parameters.add("repository.git.commandTimeout", "180");
```

Comment:
I think this is the third time I have seen this kind of configuration. Maybe put it into a common place?

Reply:
It exists four times:
- Here
- Tests
- 2x Migration Service => Temporary => Don't mind